lib/api/apiUtils/authorization/permissionChecks.js

```diff
@@ -453,39 +462,70 @@ async function validateQuotas(bucket, account, apiNames, apiMethod, inflight, lo
             }
         }
 
+        if (actionWithDataDeletion[apiMethod]) {
+            const responseTimeInNs = Number(process.hrtime.bigint() - requestStartTime);
+            monitoring.quotaEvaluationDuration.labels({
```

Comment:
No because this is called in a centralized place, and in specific APIs sometimes (copy object and multi object deletes need that, as we do not have the obj metadata at the authorization time)...